app/assets/stylesheets/pages/_show_superpower.scss

```diff
@@ -31,10 +31,52 @@
   opacity: .7;
```

Comment:
Please remove the float stuff if not in use :P

Reply:
And try to clean up a bit the html/css files, remove everything not in use anymore, and rename some classes to make sure you only style the cards/pages you worked on (and not other elements with the same class names styled by other people)